python/ray/remote_function.py

```diff
@@ -263,7 +296,13 @@ class or functions.
 
     @wrap_auto_init
     @_tracing_task_invocation
-    def _remote(self, args=None, kwargs=None, **task_options):
+    def _remote(
+        self,
+        args=None,
+        kwargs=None,
+        serialized_runtime_env_info: str = "",
```

Comment:
remove the default since we will unconditionally give a serialized_runtime_env_info

Reply:
Updated.